src/Sylius/Bundle/ShopBundle/Resources/config/services.xml

```diff
@@ -15,8 +15,11 @@
     <imports>
         <import resource="services/controller.xml" />
         <import resource="services/email.xml" />
+        <import resource="services/handlers.xml" />
         <import resource="services/listeners.xml" />
         <import resource="services/menu.xml" />
+        <import resource="services/providers.xml" />
+        <import resource="services/resolvers.xml" />
```

Comment:
Yes, thay can be moved to the CoreBundle, this will also allow to pay something via the admin for example if someone would want it.

I thought about using the existing `CoreBundle/Order` directory since it's related only to paying an order.
@NoResponseMate WDYT ?

Reply:
I'd go with a `Payment` directory since there's already one in the core's component; from what I see the `Order` one only has number generation and some promotion logic.

Reply:
It's more like a global feature allowing to pay an order with one or multiple payments, so I'm going with `OrderPay` directory instead of `Payment`.